example/src/main/java/com/spruceid/mobilesdkexample/wallet/WalletHomeView.kt

```diff
@@ -86,33 +94,72 @@ fun WalletHomeHeader(navController: NavController) {
 }
 
 @Composable
-fun WalletHomeBody(rawCredentialsViewModel: IRawCredentialsViewModel) {
+fun WalletHomeBody(
+    rawCredentialsViewModel: IRawCredentialsViewModel,
+    navController: NavController
+) {
     val scope = rememberCoroutineScope()
 
     val rawCredentials by rawCredentialsViewModel.rawCredentials.collectAsState()
 
-    if(rawCredentials.isNotEmpty()) {
-        LazyColumn(
-            Modifier
-                .fillMaxWidth()
-                .padding(top = 20.dp)
-        ) {
-            items(rawCredentials) { rawCredential ->
-                AchievementCredentialItem(
-                    rawCredential.rawCredential,
-                    onDelete = {
-                        scope.launch {
-                            rawCredentialsViewModel.deleteRawCredential(id = rawCredential.id)
+    if(!rawCredentials.isNotEmpty()) {
+        Column {
+            LazyColumn(
+                Modifier
+                    .fillMaxWidth()
+                    .padding(top = 20.dp)
+            ) {
+                items(rawCredentials) { rawCredential ->
+                    AchievementCredentialItem(
+                        rawCredential.rawCredential,
+                        onDelete = {
+                            scope.launch {
+                                rawCredentialsViewModel.deleteRawCredential(id = rawCredential.id)
+                            }
                         }
+                    ).component()
+                }
+    //        item {
+    //            vcs.map { vc ->
+    //                GenericCredentialListItems(vc = vc)
+    //            }
+    //            ShareableCredentialListItems(mdocBase64 = mdocBase64)
+    //        }
+            }
+
+            Box(
+                modifier = Modifier.fillMaxSize(),
+                contentAlignment = Alignment.BottomCenter
+            ) {
+                Button(
```

Comment:
The button is not fixed on the screen. If I add five credentials it disappears

Reply:
I think the button should be fixed on the screen now, let me know if it works on your end.